### PR TITLE
HwSettle: Fixed default value and validation annotation.

### PR DESCRIPTION
**Pull request by sselvara1**
This commit updates format of the default value in examples and the regular expression for validation annotation.

### CODE REVIEW

**Comment by yjian118** [comment]
Need to rebase the commit and resolve merge conflicts

File: examples/aio-sx-hw-settle.yaml [hunk 1]
```diff
@@ -96,7 +96,7 @@ spec:
   administrativeState: unlocked
   bootDevice: /dev/disk/by-path/pci-0000:00:0d.0-ata-1.0
   console: tty0
-  hwSettle: 0
+  hwSettle: "0"
```
Comment (yjian118) [inline examples/aio-sx-hw-settle.yaml:99]: Would prefer to use a non-zero value in the example

**Comment by yjian118** [comment]
Based on: https://github.com/Wind-River/cloud-platform-deployment-manager/blob/stable/master/api/v1/constructors.go#L808-L810, and  https://github.com/Wind-River/cloud-platform-deployment-manager/blob/stable/v2.0.10/api/v1/hostprofile_types.go#L860. I need to know if the hw_settle will be generated as "0", would prefer it to be omitted if it is zero value and keep the CRD as-is.

File: config/crd/bases/starlingx.windriver.com_hostprofiles.yaml [hunk 2]
```diff
@@ -202,7 +202,7 @@ spec:
               hwSettle:
                 description: HwSettle defines the wait time for SCSI devices to show
                   up.
-                pattern: ^[1-9][0-9]*$
```
[the remark this answers is not in the document]
Comment (sselvara1) [reply config/crd/bases/starlingx.windriver.com_hostprofiles.yaml:205]: we need to set hwSettle: "0" - this is the default value.
deploy tool generates hwSettle: "0". Not hwSettle: ""

File: examples/aio-sx-hw-settle.yaml (hunk 1 above)
Comment (sselvara1) [reply examples/aio-sx-hw-settle.yaml:99]: As discussed, default value is "0"

**Review by hugonicodemos — APPROVED** [review]
LGTM

**Review by yjian118 — APPROVED** [review]
Just a reminder to patch it back to the release stable branch

**Pull request merged**